### ISSUE TITLE
Level-up animation

### ISSUE THREAD

**Issue by davidsansome** [davidsansome/tsurukame#10]
Currently nothing happens when you level up.  We should have a satisfying level-up screen with pretty animations, and a summary of the items that you just unlocked at the next level.

**Comment by turtlemaster19** [comment]
@davidsansome, is this resolved?
![0815AE69-F731-4989-BC80-5A765CB660DC](https://user-images.githubusercontent.com/46784000/69640947-225afc80-1014-11ea-8371-81c16d5138a2.jpeg)

**Comment by turtlemaster19** [comment]
@davidsansome I agree, but I don't think any stats about the number of new items ahead should be given, but rather something simple such as, "The Crabigator has new lessons for you!"
